### ISSUE TITLE
Expected option when passing an option type, instead of type inside of option

### ISSUE THREAD

**Issue by korewaChino** [martinohmann/hcl-rs#44]
```rust
// ser.rs
#[derive(Deserialize)]
pub struct Project {
    pub proj_type: String,
    pub spec: Option<PathBuf>,
    pub dockerfile: Option<PathBuf>,
    pub scripts: Option<Vec<Script>>,
}

#[derive(Deserialize)]
pub struct Script {
    pub name: String,
    pub command: String,
}

// function to load HCL config here...
```

```hcl
// conf.hcl
project "a" {
    proj_type = "generic"
    spec = "./test.spec"
}
```

returns error
```
Message { msg: "invalid type: string \"./test.spec\", expected option", location: None }
```

**Comment by martinohmann** [comment]
Hi @korewaChino, thanks for reporting, this certainly looks like a bug. Can you provide the code you used for loading the config?

When I do

```rust
let project: Project = hcl::from_str(input).unwrap();
```

i get a different error:

```
Message { msg: "missing field `proj_type`", location: None }
```

This error is expected though since it expects a config at the top level which should look something like this:

```hcl
proj_type = "generic"
spec = "./test.spec"
```

I can reproduce your error when deserializing into this struct though:

```rust
#[derive(Deserialize)]
pub struct Config {
    #[serde(rename = "project")]
    pub projects: HashMap<String, Project>,
}

let config: Config = hcl::from_str(input).unwrap();
```

Will have a look whether this comes from.

**Comment by martinohmann** [comment]
@korewaChino I just released version 0.6.3 which includes a fix for this.
